Assets/Json/AbilityList.json

```diff
@@ -2,18 +2,114 @@
   {
     "$type": "GenericAbility, Assembly-CSharp",
     "ID": 1,
-    "Name": "GenericAbility",
-    "ImageFilePath": "GenericAbility",
-    "Description": "This is a generic ability",
-    "Cooldown": 0.0
+    "Name": "Basic Attack",
+    "ImageFilePath": "basic attack",
+    "Description": "Default melee attack.",
+    "Cooldown": 1.0
+  },
+  {
+    "$type": "GenericAbility, Assembly-CSharp",
+    "ID": 2,
+    "Name": "Single Target Melee",
+    "ImageFilePath": "single target melee",
+    "Description": "Default attack + dmg mod",
```

Comment:
T.d. hér myndi mér finnast eiga að standa:
"Deals X physical damage to a single enemy."
Þar sem X er base damage hjá Ability.

Reply:
Description er textinn sem Player mun lesa til að fá upplýsingar um hvað Abilityið gerir. Þurfum þess vegna að vera eins specific og við getum.